hack/testing/check-logs.go

```diff
@@ -91,6 +91,9 @@ func main() {
 
 	for _, record := range dat.Hits.Hits {
 		// for each message, we need to check the logs
+		if record.Fields.Message == nil {
```

Comment:
Is this necessary?  Are you seeing records without a Message field in them?  If so, that's bad, and we should investigate further.